srcpkgs/font-office-code-pro/template

```diff
@@ -0,0 +1,196 @@
+# Template file for 'font-office-code-pro'
+pkgname=font-office-code-pro
+version=1.004
+revision=1
+wrksrc="Office-Code-Pro-${version}"
+short_desc="Monospaced sans-serif font for text editors and coding environments"
+maintainer="Jony <maybe-one-day-ubermensch@protonmail.com>"
+license="OFL-1.1"
+homepage="https://github.com/nathco/Office-Code-Pro"
+distfiles="https://github.com/nathco/Office-Code-Pro/archive/${version}.tar.gz"
+checksum="9bca923d17f6c47a586d8e4567d46ccfa58fb8b8e2247b5ee2a19da1597c58f6"
+tags="monospaced sans-serif "
```

Comment:
get rid of the extra space

Reply:
I originally had it there so that when I would do var+="slashed-zero"
var turns out to be "monospaced sans-serif slashed-zero" instead of "monospaced sans-serifslashed-zero"